### PR TITLE
nixos/tests/cinnamon: init

### PR DESCRIPTION
**Pull request by bobby285271**
###### Description of changes

[I run `@ofborg test pantheon` on every Pantheon PRs](https://github.com/NixOS/nixpkgs/pulls?q=is%3Apr+%22%40ofborg+test+pantheon%22) and I want to do this on Cinnamon PRs as well.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by bobby285271** [comment]
@ofborg test cinnamon

**Comment by bobby285271** [comment]
@ofborg test cinnamon

File: pkgs/desktops/cinnamon/cinnamon-common/default.nix [hunk 1]
```diff
@@ -74,7 +74,7 @@ stdenv.mkDerivation rec {
       setproctitle
       pygobject3
       pycairo
-      python3.pkgs.xapp # The scope prefix is required
+      xapp # The scope prefix is required
```
Comment (mkg20001) [inline pkgs/desktops/cinnamon/cinnamon-common/default.nix:77]:
```suggestion
      xapp
```

Comment (bobby285271) [reply pkgs/desktops/cinnamon/cinnamon-common/default.nix:77]: I will drop 5efc2f82372840cbb3959c7bd856304e6e776bd0 very soon as it breaks the cinnamon test (which is still running and I expect ofborg will fail with timeout) :-) https://github.com/NixOS/nixpkgs/pull/189824#issuecomment-1236741751

For ofborg-eval-lib-tests failure that is #189826

Comment (mkg20001) [reply pkgs/desktops/cinnamon/cinnamon-common/default.nix:77]: ah ok

**Comment by bobby285271** [comment]
@ofborg test cinnamon

**Review by jtojnar — APPROVED** [review]
Diff LGTM.

**Pull request merged**